packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2853,7 +3013,7 @@ export default class Meeting extends StatelessWebexPlugin {
     this.locusInfo.on(LOCUSINFO.EVENTS.SELF_ADMITTED_GUEST, async (payload) => {
       this.stopKeepAlive();
       // @ts-ignore
-      if (!this.transcription && (this.config.receiveTranscription || this.receiveTranscription)) {
+      if (!this.transcription && this.config.receiveTranscription) {
```

Comment:
does this need to be changed based on what we discussed about being able to enable the transcription mid-meeting?